indico/modules/events/editing/client/js/selectors.js

```diff
@@ -199,3 +199,10 @@ export const getLastRevertableRevisionId = createSelector(
     return lastFinalRev.id;
   }
 );
+
+export const canJudgeLastRevision = createSelector(
+  getLastRevision,
+  lastRevision => {
+    return lastRevision.finalState.name !== FinalRevisionState.needs_submitter_changes;
```

Comment:
Checking for `=== FinalRevisionState.none` seems like the safer choice to me - none of the other states allow judging anyway.